File: documentoPago.cs
public class documentoPago{
	private string _idDoc;
	private tipoDocPago _docPago;
	private float _total;
	private int _idPedido;
	private Cliente _Cliente;
	private bool _registrada;

	public documentoPago(){}

	public documentoPago(string _idDoc, tipoDocPago _docPago, float _total, int _idPedido, Cliente _Cliente, bool _registrada){
		this._idDoc = _idDoc;
		this._docPago = _docPago;
		this._total =  _total;
		this._idPedido = _idPedido;
		this._Cliente = _Cliente;
		this._registrada = _registrada;
	}

	public string IdDoc{
		get{
			return _idDoc;
		}
		set{
			_idDoc = value;
		}
	}

	public tipoDocPago DocPago{
		get{
			return _docPago;
		}
		set{
			_docPago = value;
		}
	}

	public float Total{
		get{
			return _total;
		}
		set{
			_total = value;
		}
	}

	public int IdPedido{
		get{
			return _idPedido;
		}
		set{
			_idPedido = value;
		}
	}

	public Cliente Client{
		get{
			return _Cliente;
		}
		set{
			_Cliente = value;
		}
	}

	public bool Registrada{
		get{
			return _registrada;
		}
		set{
			_registrada = value;
		}
	}

}

File: java/DocumentoPago.java
enum TipoDocPago{Boleta,Factura};
public class DocumentoPago{
	private String idDoc;
	private TipoDocPago tipoDocPago;
	private float total;
	
	private int idPedido;
	
	private Cliente cliente;
	private boolean registrada;
	
	public DocumentoPago(String idDoc,TipoDocPago tipoDocPago,float total,int idPedido,Cliente cliente,boolean registrada){
		this.idDoc=idDoc;
		this.tipoDocPago=tipoDocPago;
		this.total=total;
		this.idPedido=idPedido;
		this.cliente=cliente;
		this.registrada=registrada;
	}
	
	public void setidDoc(String idDoc){
		this.idDoc=idDoc;
	}
	public String getidDoc(){
		return this.idDoc;
	}
	
    public void setEstadoPedido(TipoDocPago tipoDocPago) {
        this.tipoDocPago = tipoDocPago;
    }
	
	public TipoDocPago gettipoDocPago() {
        return tipoDocPago;
    }
	public void settotal(float total){
		this.total=total;
	}
	public float gettotal(){
		return this.total;		
	}
	public void setidPedido(int idPedido){
		//this.idPedido=idPedido;
	}
	public int getidPedido(){
		return this.idPedido;		
	}
	
	public void setcliente(Cliente cliente){
		this.cliente=cliente;
	}
	public Cliente getcliente(){
		return this.cliente;		
	}
	
	public void setregistrada(boolean registrada){
		this.registrada=registrada;
	}
	public boolean getregistrada(){
		return this.registrada;		
	}
	
	public void imprimirDocPago(){

	}

}